gdal-src/Cargo.toml

```
@@ -13,7 +13,7 @@ libsqlite3-sys = { version = "0.28.0", features = ["bundled"], optional = true }
 hdf5-src = { version = "0.8.1", optional = true, features = ["hl"] }
 hdf5-sys = { version = "0.8.1", optional = true, features = ["static"] }
 netcdf-src = { version = "0.3.3", optional = true }
-pq-src = { version = "0.1.3", optional = true }
+pq-src = { version = "0.3.1", optional = true }
```

We likely need to change this to 
```suggestion
pq-src = { version = "0.3.1", optional = true, default-features = false }
```

to disable default features